templates/CRM/Core/BillingBlock.tpl

```diff
@@ -41,7 +41,8 @@
             </div>
             <div class="content">
                 {$form.$paymentField.html}
-              {if $paymentField == 'cvv2'}{* @todo move to form assignment*}
+              {if $paymentFieldDescriptions.$name}<div class="description">{$paymentFieldDescriptions.$name}</div>
```

Comment:
@mattwire Can you move the <div> section next for better styling?

Reply:
@pradpnayak Yes, I've changed it - is that ok?